logdevice/CMake/logdevice-deps.cmake

```diff
@@ -26,30 +28,18 @@ find_package(Boost 1.55.0 MODULE
     regex
     system
     thread
-    ${_boost_py_component1}
-)
+    ${_boost_py_component})
+
 
-if(NOT Boost_FOUND)
-  message(STATUS "Boost Python Component ${_boost_py_component1} not found")
-  find_package(Boost 1.55.0 MODULE
-    COMPONENTS
-      context
-      chrono
-      date_time
-      filesystem
-      program_options
-      regex
-      system
-      thread
-      ${_boost_py_component2}
-  )
   if(NOT Boost_FOUND)
-    message(FATAL_ERROR "Boost Python Component ${_boost_py_component2} is also not found, terminating. At least one is required. ${Boost_ERROR_REASON}")
+    message(STATUS "Boost Python Component ${_boost_py_component} not found")
   else()
-    message(STATUS "Boost Python Component ${_boost_py_component2} found")
+    break()
   endif()
-else()
-  message(STATUS "Boost Python Component ${_boost_py_component1} found")
+endforeach()
+
+if(NOT Boost_FOUND)
+  message(FATAL_ERROR "Boost Python Component ${_boost_py_component2} is also not found, terminating. At least one is required. ${Boost_ERROR_REASON}")
```

Comment:
I think the message here is a bit confusing because it's only referring to the last component. Let's maybe say:

`Couldn't find any Boost python component. At least one is required, terminating.`

What do you think?

Reply:
Sure, I'll update it as you suggested